src/App.jsx

```diff
@@ -5,6 +5,8 @@ import classnames from 'classnames';
 import 'bulma/css/bulma.css';
 import './styles/App.css';
 
+const ls = require('local-storage');
```

Comment:
use import syntax: `import ls from 'local-storage';`

Reply:
and move it to the top

Reply:
done